src/main/java/com/afornalik/ox/model/player/Player.java

```diff
@@ -0,0 +1,80 @@
+package com.afornalik.ox.model.player;
+
+import com.afornalik.ox.model.board.FieldStatus;
+
+public class Player {
+
+    private final String name;
+    private int score;
+    private final FieldStatus sign;
+    private final boolean first;
```

Comment:
Should first be final? It should change with every round.

Reply:
For me I declared this variable only once entering the game. Actually i have this idea.